src/components/tooltip/Marker.js

```diff
@@ -227,16 +212,26 @@ function _markerBacktrace(
         </TooltipDetails>
       );
     default:
-      break;
+      return null;
   }
-  if ('cause' in data && data.cause) {
+}
+
+function _markerBacktrace(
```

Comment:
nit: maybe make it a stateless react component `<MaybeBacktrace>` ?

2 interesting things:
1. Make it a component, which is a bit cleaner.
2. Have the "maybe" in the name, because sometimes it outputs `null` also.

Reply:
Did it a stateless functional component and added a Maybe in front.